pxr/imaging/plugin/hdRpr/renderDelegate.cpp

```diff
@@ -311,7 +311,15 @@ VtDictionary HdRprDelegate::GetRenderStats() const {
     VtDictionary stats;
     int numCompletedSamples = m_rprApiSharedPtr->GetNumCompletedSamples();
     stats[HdPerfTokens->numCompletedSamples.GetString()] = numCompletedSamples;
-    stats[_tokens->percentDone.GetString()] = 100.0 * double(numCompletedSamples) / HdRprConfig::GetInstance().GetMaxSamples();
+
+    double percentDone = double(numCompletedSamples) / HdRprConfig::GetInstance().GetMaxSamples();
+    int numActivePixels = m_rprApiSharedPtr->GetNumActivePixels();
+    if (numActivePixels != -1) {
+        auto aovSize = m_rprApiSharedPtr->GetAovSize(m_rprApiSharedPtr->GetActiveAov());
+        int numPixels = aovSize[0] * aovSize[1];
+        percentDone = std::min(percentDone, double(numPixels - numActivePixels) / numPixels);
```

Comment:
I think this want's to be a max not a min?  because we cut off renders if numActivePixels = 0

Reply:
Yeah, you are right, I just blindly implemented your suggestion from the previous comment

Reply:
Done

Reply:
Well that's your fault for doing what I said.  :)

Reply:
Your suggestion was so obvious that it passed my trust threshold